app/templates/_package.json

```diff
@@ -14,7 +14,9 @@
     "node": ">=0.10.0"
   },
   "scripts": {
-    "test": "xo && <% if (nyc) { %>nyc <% } %>ava"
+    "test": "xo <% if (babel) { %>src/*.js test*.js && npm run babel <% } %>&& <% if (nyc) { %>nyc <% } %>ava"<% if (babel) { %>,
+    "babel": "babel src --out-dir . --presets=es2015",
```

Comment:
babel --help returns help message like below. is there another option?
-d, --out-dir [out]                  Compile an input directory of modules into an output directory